### ISSUE TITLE
Partial work : Track tag contrasts for all tracks

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3184]
This is part of the other issue https://github.com/topcoder-platform/community-app/issues/2882

Task to do:

3. Track tag contrasts for all tracks from https://github.com/topcoder-platform/community-app/issues/2882#issuecomment-516881272

The color for this will be similar to tags used in challenge listing page.

**Comment by veshu** [comment]
@rashmi73 Please check this, they are the files https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/20/files#diff-f2daa46d93e29062531cca9621bc73ff

We need to highlight when hover over those tags so please try with altering the color as highlighted below

![image](https://user-images.githubusercontent.com/2202045/62266260-59931580-b447-11e9-9aed-e2a8c7d3a408.png)

**Comment by rashmi73** [comment]
@veshu PR ready https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/23
is this expected or something else, because I am bit confused by your above comment of altering as there no much values  such that i can play with colour

**Comment by tosha5252** [comment]
Track tag colors verified changed to: 

On development: green shape with Cd and TCO text should be # 258205
On design: blue shape and TCO text should be # 0076A5
On MM: orange shape and TCO text should be #   ##BA4C00

Per comments above, these colors are taken from issue #2745 

Windows 10 env
Browser: Chrome

**Comment by tosha5252** [comment]
Dev track tag colors verified changed to:

On development: green shape with Cd and TCO text should be # 258205
On design: blue shape and TCO text should be # 0076A5
On MM: orange shape and TCO text should be # ##BA4C00
